src/GallerySwiper.jsx

```diff
@@ -1009,9 +936,24 @@ class GallerySwiper extends Component {
             }
         });
 
+        let events;
+        if (!disableArrowKeys) {
+            events = (
+                <AttachHandler
+                    target={BASE_CLASS}
```

Comment:
if you dont have a better idea i will generate https://lodash.com/docs/4.17.2#uniqueId a unique id on the component will mount event and add that to the  base_class